src/main/java/com/example/Restaurant_Management_System_REST_API/model/entity/Order.java

```diff
@@ -0,0 +1,33 @@
+package com.example.Restaurant_Management_System_REST_API.model.entity;
+
+import com.example.Restaurant_Management_System_REST_API.model.OrderStatus;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+
+@Entity
+@Getter
+@Setter
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", nullable = false)
+    private Long id;
+    private LocalDateTime orderTime;
+    private OrderStatus orderStatus;
+    @ManyToOne
+    @JoinColumn(name = "customer_id")
+    private Customer customer;
+    @ManyToOne
+    @JoinColumn(name = "table_id")
+    private Table table;
+    @ManyToMany
+    @JoinTable(
+            name = "order_menu_record",
+            joinColumns = @JoinColumn(name = "order_id"),
+            inverseJoinColumns = @JoinColumn(name = "menu_record_id"))
+    private ArrayList<MenuRecord> menuRecords;
```

Comment:
Why would you want to use specifically `ArrayList` here?

```suggestion
    private List<MenuRecord> menuRecords;
```

Reply:
my mistake! it should be List because I want my program more generic.
thanks!